styleguide/utils/console-error.js

```diff
@@ -0,0 +1,6 @@
+const original = window.console.error;
+
+window.console.error = function throwOnError(...args) {
```

Comment:
Very nice. I hope this makes failures painfully obvious in the future. 😄

Reply:
It will fail in CI! :D Yay!